src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -99,6 +99,10 @@ public partial class UpdateDurableFunction
                     instanceWatermarks);
             }
 
+            if (instanceWatermarks.Count > 0)
+            {
+                _updateMeter.UpdatedInstances.Add(instanceWatermarks.Count);
```

Comment:
Do we want to add any dimension to uniquely identify this entry?

Reply:
Added a dimension as you suggested.